.github/workflows/release-maker.yml

```
@@ -0,0 +1,56 @@
+    branches:
+      - 'main'
+
+jobs:
+  create-tag-and-release:
+    runs-on: ubuntu-latest
+    env:
+      REGEX_PATTERN: '([0-9]+\.[0-9]+(\.[0-9]+)?)$'
+    steps:
+      - name: Set Commit Message as Env Variable
+        id: set_commit_message
+        run: echo "COMMIT_MESSAGE=${{ github.event.head_commit.message }}" >> $GITHUB_ENV
```

actionlint (1.7.4)

"github.event.head_commit.message" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)